.github/workflows/deploy-storybook.yml

```diff
@@ -0,0 +1,24 @@
+name: Deploy Storybook
+on:
+  push:
+    branches: [main]
+
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  deploy:
+    name: Build & deploy
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Determine node version
+        run:
+          echo "NODE_VERSION=$(grep -oP 'node\K\d+' Dockerfile)" >> $GITHUB_ENV
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '${{ env.NODE_VERSION }}'
+          cache: yarn
```

Comment:
I'm guessing this is just quicker than using our actual container via `setup-docker-compose`?

Reply:
Yep, for frontend only stuff this is faster than going through the whole docker setup process